<?xml version="1.0" encoding="utf-8"?>
<launch>
	<node pkg="pses_basis" type="car_dashboard" name="car_dashboard" output="screen">
        <!--  allowed params: image_color, image_depth, off -->
        <param name="video_feed" value="off" />
	</node>
	<node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" > 
 		<remap from="image" to="/kinect2/hd/image_depth_rect"/>
	</node>
</launch>